tsconfig.json

```diff
@@ -20,5 +19,16 @@
 		"noFallthroughCasesInSwitch": true,
 		"noEmitOnError": true,
 		"skipLibCheck": true
-	}
+	},
+	"include": [
+		"src",
+		"test",
+		"examples",
+		"*.md",
+		"*.js",
+		".*.js"
```

Comment:
Since this config is used for eslint, it needs to include all files that we run the linter on. We're linting README.md, and the various `*.config.js`, `*rc.js` files, so need to add them here.